config.json.example

```diff
@@ -4,7 +4,8 @@
     "hostname": "avgfxawdn6c3coe3.onion",
     "key": "Il8Xas7uf6rjtc0LxYwhrx"
   },
-  "environment": "prod",
+ "environment": "prod",
+ "update_skip_delta": 28800,
```

Comment:
What's the rationale to have this value user-facing? Perhaps we can _override_ in config.json, but I think it would be more prudent to have the default be specified in `Updater.py`. Suppose that we want to change this default to a shorter period in the future, this would require an admin to configure their `config.json` and not be applied automatically.

If we ultimately decide that this config is necessary and useful, we should also validate the config.json value in https://github.com/freedomofpress/securedrop-workstation/blob/master/scripts/validate_config.py

Reply:
The rationale for doing so is to make it tweakable, per-instance and without a code push. The default is set in `sdw-launcher.py`, and is 0 sec, as the safest approach to updates is to run the update check by default. But I get your point that we might want to make it optional, and enforce a reasonable default in its absence. Happy to make the change if others agree. Will also add validation once we decide to keep it.